### DISCUSSION TITLE
Beginner question: Unexpected identifier

### DISCUSSION THREAD

**Discussion by changenowde** [akosbalasko/yarle#206] [Q&A]
Hey there! :-) 

I have node installed.
In my config json I have as templatefile: "/Users ... ExportsfromEN/config.json"

I have my template.

I start node.

Then I put the command (as describend) into the terminal.

But then I get an "unexpected idetifier" error. I don't know what to do with this.

I am really new to using the terminal, but I really want to learn how to use yarle. Is there any place you can point me to where noobs like me can find some basic understanding? That would be awesome! :-)

Thanks and greetings,
André

**Comment by akosbalasko** [comment]
Hi André,

Could you please share a bit more details about what you see? A copy-paste of the output produced by yarle would be nice. 
Here is a cool step-by-step guide on installation and usage: https://compile.blog/2021/02/25/evernote-enex-to-md/

Cheers,
akos
